server/src/test/java/org/elasticsearch/index/mapper/FieldTypeLookupTests.java

```diff
@@ -176,32 +143,6 @@ public void testRuntimeFieldsSimpleMatchToFullName() {
         }
     }
 
-    public void testRuntimeFieldsSourcePaths() {
```

Comment:
Source paths also no longer work for runtime fields - but as the comment here says, they should never be called for them anyway, as the whole point is that they're not in source.  One option might be to change the signature of `sourcePaths()` in a follow-up to take a MappedFieldType instead of a string, to ensure that we've looked up a valid field type before it gets called.

Reply:
I am wondering if the comment may be outdated, does it apply also to the script-less runtime fields? I may be getting confused.

Reply:
The reason we have this is so that a text field populated via `copy-to` can load its source to return it through the value fetcher; script-less runtime fields don't have to handle `copy-to` as they're already reading directly from source.